Tasks/AzureWebPowerShellDeployment/task.json

```diff
@@ -31,7 +31,7 @@
     {
       "name": "WebSiteName",
       "type": "pickList",
-      "label": "Web App Name",
+      "label": "Web App Name with out slot.If your web app uses slot, mention in **Slot** field, else default **Production** slot is considered",
```

Comment:
Actually. No..it is required. Websitenames with slot would have.. test(slot), I was suggesting instead of that...have websitename as just 'test' and put the slot in slot field.